microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension/install.md

```diff
@@ -235,3 +235,7 @@ Next, see [Get started using the DevTools extension for Visual Studio Code](./ge
 * [Get started using the DevTools extension for Visual Studio Code](./get-started.md)
 * [Opening DevTools and the DevTools browser](./open-devtools-and-embedded-browser.md)
 * [Microsoft Edge DevTools extension for Visual Studio Code](../microsoft-edge-devtools-extension.md)
+* [Sample code for DevTools](../../devtools-guide-chromium/sample-code/sample-code.md)
+* [Download the WebView2Samples repo](../../webview2/how-to/machine-setup.md#download-the-webview2samples-repo) in _Set up your Dev environment for WebView2_.
+   * [Clone the WebView2Samples repo](../../webview2/how-to/machine-setup.md#clone-the-webview2samples-repo) in _Set up your Dev environment for WebView2_.
```

Comment:
How come we talk about WebView2 in this VS Code DevTools extension doc? I don't see the link here.

Reply:
removed wv2 lines